src/components/ui/QuestionBoard.js

```diff
@@ -189,10 +198,11 @@ class QuestionBoard extends React.Component {
                 <ButtonWrapper>
                     {questions[count].Answers.map((answer, i) => {
                         if (answer) {
+                            const key = this.getButtonKey(i);
                             return (
-                                <AnswerButton key={`${i}`}
-                                    onClick={() => this.onButtonClick(i,questions[count].Id)}>
-                                        {questions[count].Answers[i]}
+                                <AnswerButton key={key}
+                                    onClick={() => this.onButtonClick(i, questions[count].Id, key)}>
```

Comment:
Coding convention
Please make onClick moving underneath the key

Reply:
Done!